docs/data/data-grid/server-side-data/lazy-loading.md

```diff
@@ -57,37 +76,37 @@ Use the `lazyLoadingRequestThrottleMs` prop to set a custom time, as shown below
 A new page is loaded when the scroll reaches the bottom of the viewport area.
 
 You can use the `scrollEndThreshold` prop to change the area that triggers new requests.
 
 The demo below shows how infinite loading mode works.
-Page size is set to `15` and the mock server is configured to return a total of 100 rows.
+The page size is set to `15` and the mock server is configured to return a total of 100 rows.
 When the response contains no new rows, the Grid stops requesting new data.
 
 {{"demo": "ServerSideLazyLoadingInfinite.js", "bg": "inline"}}
 
-## Updating the loading mode
+### Changing the loading mode
 
-The Grid changes the loading mode dynamically if the total row count gets updated by changing the `rowCount` prop, returning different `rowCount` in `GridGetRowsResponse` or via `setRowCount()` API.
+The Grid changes the loading mode dynamically if the total row count gets updated by changing the `rowCount` prop, returning a different `rowCount` in `GridGetRowsResponse`, or via the `setRowCount()` method.
 
-Based on the previous and the new value for the total row count, the following scenarios are possible:
+Depending on the previous and new values for the total row count, the following scenarios are possible:
 
-- **Unknown `rowCount` to known `rowCount`**: When the row count is set to a valid value from an unknown value, the Data Grid switches to viewport loading mode. It checks the number of already fetched rows and adds skeleton rows to match the provided row count.
+- **Unknown to known row count** – When the row count is set to a valid value from an unknown value, the Data Grid switches to viewport loading mode. It checks the number of already fetched rows and adds skeleton rows to match the provided row count.
 
-- **Known `rowCount` to unknown `rowCount`**: If the row count is updated and set to `-1`, the Data Grid resets, fetches the first page, then sets itself to infinite loading mode.
+- **Known to unknown row count** – If the row count is updated and set to `-1`, the Data Grid resets, fetches the first page, then sets itself to infinite loading mode.
 
-- **Known `rowCount` greater than the actual row count**: This can happen either by reducing the value of the row count after more rows were already fetched, or if the row count was unknown and the Grid (while in the infinite loading mode) already fetched more rows. In this case, the Grid resets, fetches the first page, and then continues in one mode or the other depending on the new value of the `rowCount`.
+- **Known row count greater than the actual row count** – This can happen either by reducing the value of the row count after more rows were already fetched, or if the row count was unknown and the Grid already fetched more rows while in infinite loading mode. In this case, the Grid resets, fetches the first page, and then continues in one mode or the other depending on the new value of the `rowCount`.
```

Comment:
Since we don't use backticks in any other places.
```suggestion
- **Known row count greater than the actual row count** – This can happen either by reducing the value of the row count after more rows were already fetched, or if the row count was unknown and the Grid already fetched more rows while in infinite loading mode. In this case, the Grid resets, fetches the first page, and then continues in one mode or the other depending on the new value of the row count.
```